generators/generatorPickerFunctions.cc

```diff
@@ -471,3 +471,57 @@ ostream& polynomialMassAndTPrimeSlopePicker::print(ostream& out) const {
 	out << "    t' slopes polynomial ... " << _tPrimeSlopePolynomial.GetExpFormula() <<endl;
 	return out;
 }
+
+
+bool
+rpwa::uniformMassAndTPicker::init(const libconfig::Setting& setting)
+{
+	if (not massAndTPrimePicker::initTPrimeAndMassRanges(setting)) {
+		printErr<< "could not initialize t' or mass range settings in 'uniformMassExponentialT'." << endl;
+		return false;
+	}
+	if(not setting.exists("tPrimeMin") or not setting.exists("tPrimeMax")) {
```

Comment:
probably one should also check the types here, setting tPrimeMin to a list still mean it exists, and initTPrimeAndMassRanges would simply print a warning and use some default